src/tracing.js

```diff
@@ -1,29 +1,27 @@
 import { AwsInstrumentation } from "@opentelemetry/instrumentation-aws-sdk";
 import { Resource } from "@opentelemetry/resources";
-import { NodeTracerProvider } from "@opentelemetry/sdk-trace-node";
 import {
-  ConsoleSpanExporter,
-  SimpleSpanProcessor,
+	ConsoleSpanExporter,
```

Comment:
this is a tab space? 😨

Reply:
Looks like biome uses tabs by default.

I re-introduced `.editorconfig` and instructed biome to use that instead
[`a75287d` (#10)](https://github.com/trivikr/aws-sdk-js-otel-test/pull/10/commits/a75287df9b730e3f65ac9d2d4ddb4abf04c0ed3a)